src/frontend/src/lib/components/hero/Hero.svelte

```diff
@@ -51,12 +51,22 @@
 		}
 
 		&.chainfusion {
-			background: radial-gradient(
-				90.18% 135.69% at 50% 135.69%,
-				#3653cb 0%,
-				#5331a6 40.06%,
-				#191e86 100%
-			);
+      background: #03020B;
+
+			position: relative;
+
+      &::before {
```

Comment:
We cannot use a unique background anymore?

Can you provide screenshots from various devices as the before contains fixed values?

Reply:
Well, since they are cogitating the idea of having some smaller animation there, I found it easier to put the ellipse. And it was easier to code replicating from Figma eheh!

Reply:
A few screenshots

<img width="347" alt="Screenshot 2024-07-18 at 12 58 45" src="https://github.com/user-attachments/assets/8d11f11f-8973-4c80-86df-8e38a097478c">
<img width="414" alt="Screenshot 2024-07-18 at 12 58 54" src="https://github.com/user-attachments/assets/5ee24319-0ef6-42fb-ad09-c9e9eb79c0cb">
<img width="409" alt="Screenshot 2024-07-18 at 12 59 02" src="https://github.com/user-attachments/assets/0e6e1f9e-f980-4070-9364-e118cf58dacc">
<img width="427" alt="Screenshot 2024-07-18 at 12 59 09" src="https://github.com/user-attachments/assets/92906c65-7a50-461a-965e-3c81eeaf5ce0">
<img width="374" alt="Screenshot 2024-07-18 at 12 59 17" src="https://github.com/user-attachments/assets/ef318181-00c8-4c94-985e-25f78a669dbb">
<img width="357" alt="Screenshot 2024-07-18 at 12 59 26" src="https://github.com/user-attachments/assets/354f8642-0527-4e82-8302-a2ce4ec7f10b">

Reply:
Nice! on very large screen it's fine as well?

Reply:
it was not so nice, so i corrected it to adapt to larger screens (above 1160px that is the fixed width that we had up until now)

before:
`width: 1160px;`

after:
`width: max(100%, 1160px);`


here the results

<img width="3289" alt="Screenshot 2024-07-18 at 13 57 36" src="https://github.com/user-attachments/assets/7a73dbe5-6475-4230-a9d1-464cee312434">

Reply:
Nice! Thx